examples/data-sources/cockroach_folder/data-source.tf

```diff
@@ -0,0 +1,11 @@
+variable "prod_folder_id" {
+  type = string
+}
+
+data "cockroach_folder" "team1" {
+  path = "/prod/team1"
+}
+
+data "cockroach_folder" "prod" {
```

Comment:
I really like the design of this data source - the pathing is nice and intuitive!